src/components/theme/light-theme.css

```diff
@@ -1,25 +1,42 @@
 /* Default and primary button colors differ within modals */
-.light-theme :global(.ant-btn-default) {
-    background-color: var(--light-button-default-background);
-    border-color: var(--light-theme-modal-btn-default-color);
-    color: var(--light-button-default-color);
+
```

Comment:
updated to match [this styling](https://xd.adobe.com/view/5f448466-eebd-4ad2-bdd8-314a55133403-0a17/screen/63079c1e-f6f9-44d8-a235-39e10d40acbe/)